QUBEKit/ligand.py

```diff
@@ -222,6 +222,11 @@ def __init__(self, mol_input, name=None):
         self.bond_types = None
         self.angle_types = None
         self.dihedral_types = None
+        self.improper_types = None
+
+        self.dih_start = {}
```

Comment:
maybe something like self.scan_settings = {} then have a named tuple (start, end, increment) then index it by the torsion we are scanning. Then we can look through the torsions and get the settings instead of 3 different dicts?